kstool/kstool.cpp

```diff
@@ -77,13 +77,14 @@ int main(int argc, char **argv)
     ks_engine *ks;
     ks_err err = KS_ERR_ARCH;
     char *mode, *assembly = NULL;
+    uint64_t start_addr;
```

Comment:
it is better to initialize `start_address = 0` here, so it becomes default value, even when `argc == 2` (which you are missing now). then `start_address = 0` can be removed.